File: Data Structures & Algorithms/longest-consecutive-sequence/submission-2.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        unique = set()
        for num in nums:
            unique.add(num)
        
        res = 0
        for num in unique:
            if num - 1 not in unique:
                cur = num
                curCount = 0
                while cur in unique:
                    curCount += 1
                    cur += 1
                    res = max(res, curCount)
        
        return res

File: Data Structures & Algorithms/kth-largest-element-in-an-array/submission-2.java
class Solution {
    public int findKthLargest(int[] nums, int k) {
        int[] arr = new int[2001];
        for (int num : nums) {
            arr[num + 1000]++;
        }

        int count = 0;
        for (int i = arr.length - 1; i >= 0; i--) {
            count += arr[i];
            if (count >= k) {
                return i - 1000;
            }
        }
        return -1;
    }
}
